integrations/pennylane/helpers.ts

```diff
@@ -0,0 +1,173 @@
+import type {
+    CreateInvoice,
+    CreateSupplier,
+    NangoAction,
+    PennylaneCustomer,
+    PennylaneIndividualCustomer,
+    PennylaneInvoice,
+    PennylaneSupplier
+} from '../models.js';
+
+export function toInvoice(invoice: PennylaneInvoice): PennylaneInvoice {
```

Comment:
for consistency purposes these mapping functions should go in specific files

`mappers/to-invoice.ts`
`mappers/to-customer.ts`
`mappers/to-supplier.ts`